src/domains/gps-common/00_monitor.tf

```diff
@@ -21,3 +21,36 @@ data "azurerm_monitor_action_group" "email" {
   resource_group_name = var.monitor_resource_group_name
   name                = local.monitor_action_group_email_name
 }
+
+##Alert
+resource "azurerm_monitor_scheduled_query_rules_alert" "payments_gpd_inconsistency_error" {
+  count = var.env_short == "p" ? 1 : 0
```

Comment:
it right only prod ?

Reply:
Yes, we want to have a periodic alert only in prod. For dev and uat environment it is not mandatory to have a constant check, a manual check is sufficient